linen_examples/pixelcnn/train.py

```diff
@@ -220,8 +220,7 @@ def train():
       'params': init_rng,
       'dropout': dropout_rng
   }, init_batch)['params']
-  optimizer_def = optim.Adam(
-      learning_rate=FLAGS.learning_rate, beta1=0.95, beta2=0.9995)
+  optimizer_def = optim.Adam(beta1=0.95, beta2=0.9995)
```

Comment:
Wait, why did you remove the learning rate here?

Reply:
Because it's over-ridden on [this line](https://github.com/google/flax/blob/4dcad3a1953f37e0a519bdefbe1867f6100d3edb/examples/pixelcnn/train.py#L144), so unnecessary to feed it in here.

Reply:
Oh right, makes sense.